addons/common/functions/fnc_showHud.sqf

```diff
@@ -42,15 +42,15 @@ if (_reason != "") then {
     _reason = toLower _reason;
     if (_mask isEqualTo []) then {
         TRACE_2("Removing", _reason, _mask);
-        [GVAR(showHudHash), _reason] call CBA_fnc_hashRem;
+        GVAR(showHudHash) deleteAt _reason;
     } else {
         while {(count _mask) < 10} do { _mask pushBack true; };
         TRACE_2("Setting", _reason, _mask);
-        [GVAR(showHudHash), _reason, _mask] call CBA_fnc_hashSet;
+        GVAR(showHudHash) set [_reason, _mask];
     };
 };
 
-GVAR(showHudHash) params ["", "_reasons", "_masks"];
+(toArray GVAR(showHudHash)) params ["_reasons", "_masks"];
```

Comment:
The code actually only uses reasons for logging.
Its a waste to always copy them out even though you only use it inside the logging.
Just use `keys GVAR(showHudHash)` down below in the TRACE_2 instead of always pulling them out even though TRACE is normally disabled.